analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -34,9 +34,9 @@
 using SonarAnalyzer.LiveVariableAnalysis.CSharp;
 using SonarAnalyzer.Rules.SymbolicExecution;
 using SonarAnalyzer.SymbolicExecution;
-using SonarAnalyzer.SymbolicExecution.Roslyn.RuleChecks.CSharp;
 using SonarAnalyzer.SymbolicExecution.Sonar;
 using StyleCop.Analyzers.Lightup;
+using RuleChecks = SonarAnalyzer.SymbolicExecution.Roslyn.RuleChecks.CSharp;
```

Comment:
The alias can be removed after the old NullPointerDereference is removed from SonarRules.

Reply:
I don't think we'll be able to remove the usage. Let's do it the other way around - keep the new version clean, and the old engine noisy. So it became unused once we completely remove the engine.

Let's move the old files to `SymbolicExecution/Sonar/Rules` directory and namespace. And create alias for them.

Reply:
> Let's move the old files to SymbolicExecution/Sonar/Rules directory and namespace. And create alias for them.

Baby steps in the next PR or in this one?

Reply:
As you wish

Reply:
Decision: Place it in `SonarAnalyzer.CSharp\SymbolicExecution\Sonar\Analyzers` (follow up PR).